### PR TITLE
Add Jenkins build 

### PR DESCRIPTION
**Pull request by snehaso**
runs bundle, tests and publishes gem to https://rubygems.org

https://jenkins.eng.fre.ag/job/FreeAgent/job/hermod/job/setup_ci/11

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.3.1
```
Comment (alyssais) [inline .ruby-version]: Why 2.3.1?

Comment (snehaso) [reply .ruby-version]: Hermod requires an older version of bundler 1.6.1. I started with using 2.6.3 but it gave bundler error, so switched to 2.3.1. 

Available versions are:
https://github.com/fac/puppet-config/blob/production/role_modules/jenkins_ci_role/manifests/rubies.pp

Comment (snehaso) [reply .ruby-version]: my bad. changed it to 2.6

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+2.6.0
```
Comment (alyssais) [inline .ruby-version]: Actually, should this be 2.6.3? Is there a reason for not using the latest version?

Comment (snehaso) [reply .ruby-version]: @alyssais  Ruby 2.6.3 looks for bundler version >2.0. We can change bundler version in gemspec. https://github.com/fac/hermod/blob/master/hermod.gemspec#L28

I'm not sure why Hermod uses older version of bundler.

Comment (snehaso) [reply .ruby-version]: upgraded to bundler 2 and ruby 2.6.3.

File: lib/hermod/version.rb [hunk 3]
```diff
@@ -1,3 +1,3 @@
 module Hermod
-  VERSION = "2.4.1"
```
Comment (spdawson) [inline lib/hermod/version.rb]: Should this be `2.5.0` to match what's mentioned in the changelog?

Comment (snehaso) [reply lib/hermod/version.rb]: Done

**Pull request merged**